pkg/operator/rbac.go

```diff
@@ -81,7 +81,7 @@ func MakeScopedRBACObjects(fbName, fbNamespace string, saAnnotations map[string]
 
 	rb := rbacv1.RoleBinding{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      "kubesphere:fluent",
+			Name:      fmt.Sprintf("fluent-operator-%s", fbName),
```

Comment:
Maybe we can follow MakeRBACObjects' convention:
```suggestion
			Name:      fmt.Sprintf("fluent-operator-fluent-bit-%s", fbName),
```